hack/test-cmd.sh

```diff
@@ -99,6 +99,8 @@ os::cmd::try_until_success "oc get service kubernetes --namespace default --conf
 os::cmd::try_until_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority ${MASTER_CONFIG_DIR}/server-ca.crt -u test-user -p anything" $(( 160 * second )) 0.25
 # wait for the CRD to be available
 os::cmd::try_until_success "oc get clusterresourcequotas --config='${ADMIN_KUBECONFIG}'" $(( 160 * second )) 0.25
+os::cmd::try_until_success "oc get scc --config='${ADMIN_KUBECONFIG}'" $(( 160 * second )) 0.25
```

Comment:
This is already checked in https://github.com/openshift/origin/blob/7e95f8675d2c38c7fc7c53b3c9b9bb1e5e1b3855/hack/local-up-master/lib.sh#L252-L261

Please line up the checks in `test-cmd` and `test-integration` so that they appear in the same spot - pick either of "right after KAS start" or "once all is set up" (there's a wait for CRQs in `test-integration` as wel AFAIR). I don't think the spot matters now that we don't need the CRD to be available for OAS bootstrap.

Reply:
This can be done as a follow-up if we manage to get the tests passing.